OpenRA.Game/Graphics/TargetLineRenderable.cs

```diff
@@ -42,22 +46,22 @@ public void Render(WorldRenderer wr)
 			if (!waypoints.Any())
 				return;
 
-			var iz = 1 / wr.Viewport.Zoom;
+			var iz = width / wr.Viewport.Zoom;
```

Comment:
nit: `iz` was short for "inverse zoom", so should be renamed now that this is a scaled width.

Reply:
Fixed.